metabox/README.md

```diff
@@ -0,0 +1,84 @@
+# Introduction
+
+[Checkbox] is used on a wide variety of devices (laptops, servers, IoT), with a variety of releases (18.04, 20.04, etc.), in a few different ways (run locally, using Checkbox remote). Over the years, manually testing Checkbox before each stable release has become more and more difficult.
+
+Comes Metabox.
```

Comment:
?

Reply:
Sorry, broken Frenglish. I updated that.